components/core/CMakeLists.txt

```diff
@@ -234,15 +262,27 @@ set(SOURCE_FILES_unitTest
         src/clp/streaming_archive/writer/Segment.hpp
         src/clp/streaming_archive/writer/utils.cpp
         src/clp/streaming_archive/writer/utils.hpp
+        src/clp/streaming_compression/Compressor.hpp
+        src/clp/streaming_compression/Constants.hpp
+        src/clp/streaming_compression/Decompressor.hpp
+        src/clp/streaming_compression/passthrough/Compressor.cpp
+        src/clp/streaming_compression/passthrough/Compressor.hpp
+        src/clp/streaming_compression/passthrough/Decompressor.cpp
+        src/clp/streaming_compression/passthrough/Decompressor.hpp
+        src/clp/streaming_compression/zstd/Compressor.cpp
+        src/clp/streaming_compression/zstd/Compressor.hpp
+        src/clp/streaming_compression/zstd/Constants.hpp
+        src/clp/streaming_compression/zstd/Decompressor.cpp
+        src/clp/streaming_compression/zstd/Decompressor.hpp
+        src/clp/StringReader.cpp
+        src/clp/StringReader.hpp
         src/clp/Utils.cpp
```

Comment:
What's the naming policy of util files? All other names are in lower case except this one.

Reply:
Yeah, it should be lowercase, but we haven't completely refactored clp yet. I would rather do that in another PR to avoid making the diff harder to review.